src/containers/shared/metaParser.tsx

```diff
@@ -10,8 +10,26 @@ export function getAuthAccounts(tx: any) {
   return tx.AuthAccounts?.map((acc: any) => acc?.AuthAccount?.Account)
 }
 
+export function findEffectivePrice(tx: any) {
```

Comment:
Why is this method even necessary? Why not just do `tx.EPrice`?

Reply:
if tx.eprice changes itll be easier to manage.

Reply:
What do you mean if `tx.EPrice` changes? It's not likely to change, or change its format. This is additional code that must be maintained.

Reply:
well we've had a bunch of fields change in the past month or so - after the AMM stuff is finalized its less likely to change.

In terms of maintenance - there's nothing to maintain here. this function simply reformats the fields and returns them. Other functions in this class do the same thing.

Reply:
The function is unnecessary - it's really not doing anything that just using `tx.EPrice` isn't doing.

Reply:
I removed this in favor of formatAmount since it works as expected here.